packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -379,7 +379,7 @@ export class UUIColorPickerElement extends LabelMixin('label', LitElement) {
     this.hue = h;
     this.saturation = s;
     this.lightness = l;
-    this.alpha = this.opacity ? a * 100 : 100;
+    this.alpha = this.opacity != undefined ? a * 100 : 100;
```

Comment:
Not sure how `opacity` can be undefined? It's a boolean with `false` as default value.

In the story it set `value: undefined`.